src/azul/chalice.py

```diff
@@ -80,6 +81,14 @@ class ServiceUnavailableError(ChaliceViewError):
     STATUS_CODE = 503
 
 
+@attr.s(auto_attribs=True, frozen=True, kw_only=True)
+class MetricThreshold:
+    lambda_name: str
+    handler_name: Optional[str] = attr.ib(default=None)
+    metric: str
```

Comment:
Document where the valid values for this come from (link to AWS docs) and defined them in a static class attribute.